gradle.properties

```diff
@@ -1,12 +1,12 @@
 projectVersion=4.2.1-SNAPSHOT
```

Comment:
Can you bump this to a new major version

Reply:
@graemerocher yes, sure. Already done